src/components/cluster/detail/upgrade_cluster_modal.js

```diff
@@ -76,57 +77,33 @@ class UpgradeClusterModal extends React.Component {
             if (diffEdit.kind === 'E') {
               let component = components[diffEdit.path[0]];
               return (
-                <div
-                  className='release-selector-modal--component'
-                  key={component.name}
-                >
-                  <span className='release-selector-modal--component--name'>
-                    {component.name}
-                  </span>
-                  <span className='release-selector-modal--component--version'>
-                    <span className='lhs'>
-                      <span>{diffEdit.lhs}</span>
-                      <span />
-                    </span>{' '}
-                    <span className='rhs'>{diffEdit.rhs}</span>
-                  </span>
-                </div>
+                <ReleaseComponentLabel
+                  name={component.name}
+                  oldVersion={diffEdit.lhs}
+                  version={diffEdit.rhs}
+                />
               );
             }
 
             if (diffEdit.kind === 'N') {
               let component = diffEdit.rhs;
               return (
-                <div
-                  className='release-selector-modal--component'
-                  key={component.name}
-                >
-                  <span className='release-selector-modal--component--name'>
-                    {component.name}
-                  </span>
-                  <span className='release-selector-modal--component--version'>
-                    <span className='rhs'>{component.version} (added)</span>
-                  </span>
-                </div>
+                <ReleaseComponentLabel
+                  isAdded={true}
+                  name={component.name}
+                  version={component.version}
+                />
               );
             }
 
             if (diffEdit.kind === 'D') {
               let component = diffEdit.lhs;
               return (
-                <div
-                  className='release-selector-modal--component'
-                  key={component.name}
-                >
-                  <span className='release-selector-modal--component--name'>
-                    {component.name}
-                  </span>
-                  <span className='release-selector-modal--component--version'>
-                    <span className='lhs'>
-                      <span>{component.version}</span> <span>(removed)</span>
-                    </span>
-                  </span>
-                </div>
+                <ReleaseComponentLabel
+                  isRemoved={true}
```

Comment:
you can also do it as `isRemoved` there is no need for `={true}`

Reply:
Very nice, thanks!